com.jetbrains.RustRover.yml

```diff
@@ -0,0 +1,140 @@
+app-id: com.jetbrains.RustRover
+command: rustrover
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - --allow=devel
+  - --device=all
```

Comment:
why does this needs 'device=all' ?

Reply:
https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Community/commit/32b7b59ff29f5b53329227a178cee8bcb08ff94b

Reply:
hmm not sure if really required here. Was using other 'Jetbrains' flatpaks as reference for this one. Maybe `device=dri` is sufficient here...

Reply:
on a second thought though it might require this additional permission for the 'CodeWithMe' functionality, which enables pair programming sessions in the IDE: https://www.jetbrains.com/code-with-me/

Reply:
> https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Community/commit/32b7b59ff29f5b53329227a178cee8bcb08ff94b

Does this app have those functionalities too? Did you check? Otherwise this is not relevant here.

Reply:
Did you saw my previous comment and checked the website? I guess device access like sound cards and web cams is required for this feature to work inside the app.

Reply:
I'm asking if they share the same features. What Intellij IDEA is doing might not be relevant for this one too.

Reply:
What about those:

![image](https://github.com/user-attachments/assets/b4162d7b-1b8d-4c12-b26f-41f0f06163bc)

Anyway if this is a blocker, we can restrict to `dri` only and waiting for the first user to complain and raise a ticket.... 

Btw: I don't really understand the rules and guides on how those things are granted or not. Something that's accepted by ANY other Jetbrains flatpak is put on question in this one.... why did it get granted in the others?